Packs/Securonix/Integrations/Securonix/README.md

```diff
@@ -2,6 +2,11 @@
 Use the Securonix integration to manage incidents and watchlists.
 Integration was build and tested with SNYPR Version 6.3.
```

Comment:
do we know if this is the version for both cloud and on-prem ?

Reply:
No, as we get access to  cloud base only. I know for a fact that it is working for an on prem of version 6.2 as well.

Reply:
according to the vendor, API should be compatible for both deployments.